.github/pull_request_template.md

```diff
@@ -1,25 +1,36 @@
-# Description
+## Description
 
-Please include a summary of the change.
+*Replace this paragraph with a description of what this PR is doing. If you're modifying existing behavior, describe the existing behavior, how this MR is changing it, and what motivated the change.*
```

Comment:
In particular, if this is a breaking change, specify this here and what APIs where changed.